ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -87,6 +95,20 @@ - (RCTBridge *)bridge {
     }];
 }
 
+RCT_REMAP_METHOD(isDataCollectionEnabled,
+                 isDataCollectionEnabled_resolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject) {
+
+    resolve(@([UAirship shared].isDataCollectionEnabled));
+}
+
+RCT_REMAP_METHOD(isPushTokenRegistrationEnabled,
+                 isPushTokenRegistrationEnabled_resolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject) {
+
+    resolve(@([UAirship push].pushTokenRegistrationEnabled));
+}
+
```

Comment:
Doesn't iOS need `setDataCollectionEnabled()` and `setPushTokenRegistrationEnabled()`?

Reply:
Its like 67 and 71

Reply:
@BrianBatchelder It's like the existing implementation 
```
RCT_EXPORT_METHOD(setBadgeNumber:(NSInteger)badgeNumber) {
    [[UAirship push] setBadgeNumber:badgeNumber];
}
```

Reply:
OK, weird. I must have missed those two methods. Sorry about that!